### PR TITLE
Fix 'Reset dashboard' button xpath and mark it as blocked

### PR DESCRIPTION
**Pull request by vrutkovs**
{{pytest: cfme/tests/test_csrf.py -v }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit fe8e01ee9a1c5323b90dd2e4dfbe1c62ca9ea98a:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit be3e8cc7c6d2094e3b801c872eed6311d5500cf1:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 80a40c90d6ce502cecf2312299d3eb1e855c7efb:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/test_csrf.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 from utils.wait import wait_for, TimedOutError
 
 
+@pytest.mark.meta(blockers=['GH#ManageIQ/manageiq:5657'])
+@pytest.mark.meta(blockers=[1285778])
```
Comment (mfalesni) [inline cfme/tests/test_csrf.py]: @pytest.mark.meta(blockers=[1285778, 'GH#ManageIQ/manageiq:5657'])

**Comment by dajoRH** [comment]
Lint report for commit f20fc85c4997bb18665c15e4d1ce3092c6aa86ff:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 62f414a8a7c25e02f2db11f67c5906f6a24726b7:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e6c15494173651b1f0a154e30100a38a2e0d05dd:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8a40fc316e2d5f1ef687c8055f64d720ee336317:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**